habit/src/actions/login.js

```diff
@@ -0,0 +1,20 @@
+import Login from "../containers/login/LoginScreen";
+
+export const login = (data) => {
+	let { email, password } = data;
+	email = email.toLowerCase();
+	return (dispatch) => {
+		return fetch('/login', {
+			method: 'POST',
+			headers: {  //TODO AYUSHI change them according to the API
+				Accept: 'application/json',
+				'Content-Type': 'application/json',
+			},
+			body: JSON.stringify(email),
```

Comment:
Why is this email? shouldn't it be data?

Reply:
yes, fixes are in the next commit